message.go

```diff
@@ -1,11 +1,15 @@
 package stream_chat // nolint: golint
 
 import (
+	"encoding/json"
 	"errors"
 	"net/http"
 	"net/url"
 	"path"
 	"time"
+
+	jlexer "github.com/mailru/easyjson/jlexer"
```

Comment:
why rename ?

Reply:
Not sure, it is what go imports did by default, and I didn't fight it.

Reply:
It's unneeded, might be outdated goimports.